### PR TITLE
enable cloud test

### PR DESCRIPTION
**Pull request by yasserfaraazkhan**
```release-note
none
```
1. Notify admin 
requirements to run locally (server changes)
- make test-data in server
- MM_NOTIFY_ADMIN_COOL_OFF_DAYS= 0.00000001 (1 mili second or any milliseconds)
Screenshot 2022-11-17 at 4 40 33 PM

2. Subscriptions section
Screenshot 2022-11-17 at 4 43 08 PM

3. Payment Information section
Screenshot 2022-11-17 at 4 44 29 PM

4. Subscription free trial
Screenshot 2022-11-17 at 5 57 13 PM

5. Billing History

Screenshot 2022-11-21 at 6 36 35 AM

6. Cloud pricing modal
Screenshot 2022-11-21 at 7 29 31 AM

### CODE REVIEW

**Comment by yasserfaraazkhan** [comment]
/e2e-test TEST_FILTER='--stage="@prod" --excludeGroup="@te_only,@high_availability"'

**Comment by yasserfaraazkhan** [comment]
/e2e-test TEST_FILTER="--stage='@prod' --excludeGroup='@te_only,@high_availability'"

**Comment by yasserfaraazkhan** [comment]
/e2e-test TEST_FILTER="--stage:'@prod' --excludeGroup:'@te_only,@high_availability'"

File: e2e/cypress/tests/integration/enterprise/cloud/billing/billing_history_free_trial_spec.ts [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        cy.get('@tableRows').eq(2).find('div.BillingHistory__paymentStatus').should('have.text', 'Paid');
+    });
+
+    it('MM-T3491_2 Validate the contents of downloaded PDF invoice', () => {
+        cy.get('tr.BillingHistory__table-row').as('tableRows');
+
+        // * Check for default record's length in grid
+        cy.get('@tableRows').should('have.length', 3);
+
+        // * Check the invoice lins
```
Comment (furqanmlk) [inline e2e/cypress/tests/integration/enterprise/cloud/billing/billing_history_free_trial_spec.ts:162]: Can you check the spellings ?

Comment (yasserfaraazkhan) [reply e2e/cypress/tests/integration/enterprise/cloud/billing/billing_history_free_trial_spec.ts:162]: done Furqan

File: e2e/cypress/tests/integration/enterprise/cloud/billing/billing_history_free_trial_spec.ts [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+                id: 'in_1Lz8b0I67GP2qpb43kcuMyFP',
+                number: '8D53267B-0006',
+                create_at: 1667263490000,
+                total: 65,
+                tax: 0,
+                status: 'open',
+                description: '',
+                period_start: 1664582400000,
+                period_end: 1667260800000,
+                subscription_id: 'prod_K0AxuWCDoDD9Qq',
```
Comment (AGMETEOR) [inline e2e/cypress/tests/integration/enterprise/cloud/billing/billing_history_free_trial_spec.ts:27]: Maybe we should use something like `sub_...`

File: e2e/cypress/tests/integration/enterprise/cloud/billing/billing_history_free_trial_spec.ts [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+                id: 'in_1LntnKI67GP2qpb4VObu3NgP',
+                number: '8D53267B-0005',
+                create_at: 1664584986000,
+                total: 733,
+                tax: 0,
+                status: 'failed',
+                description: '',
+                period_start: 1661990400000,
+                period_end: 1664582400000,
+                subscription_id: 'prod_K0AxuWCDoDD9Qq',
```
Comment (AGMETEOR) [inline e2e/cypress/tests/integration/enterprise/cloud/billing/billing_history_free_trial_spec.ts:49]: Use `sub_...`

File: e2e/cypress/tests/integration/enterprise/cloud/billing/billing_history_free_trial_spec.ts [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+                    },
+                ],
+                current_product_name: 'Cloud Professional',
+            },
+        ],
+    });
+    cy.intercept('GET', '**/api/v4/cloud/subscription', {
+        statusCode: 200,
+        body: {
+            id: 'sub_test1',
```
Comment (AGMETEOR) [inline e2e/cypress/tests/integration/enterprise/cloud/billing/billing_history_free_trial_spec.ts:88]: For consistency, let this id match the subscription id in the invoices above

Comment (yasserfaraazkhan) [reply e2e/cypress/tests/integration/enterprise/cloud/billing/billing_history_free_trial_spec.ts:88]: Got it Allan. Updated the changes

**Review by AGMETEOR — APPROVED** [review]

**Review by furqanmlk — COMMENTED** [review]
@yasserfaraazkhan 
Can you please fix the conflict ?

**Comment by yasserfaraazkhan** [comment]
> @yasserfaraazkhan
> Can you please fix the conflict ?

@furqanmlk Done

**Pull request merged**